### PR TITLE
feat(order-summary): add simple order summary component

### PR DESCRIPTION
**Pull request by tw15egan**
## Order Summary

Adds the Order Summary component, still need to add the category functionality / tests, but wanted to get it up so people can get some eyes on it!

### To Do
- [x] Add category functionality
- [x] Tests

### CODE REVIEW

File: .storybook/components/OrderSummaryStory.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+} from '../../components/OrderSummary';
+import Button from '../../components/Button';
+import Dropdown from '../../components/Dropdown';
+import DropdownItem from '../../components/DropdownItem';
+
+storiesOf('OrderSummary', module)
+  .addWithInfo(
+    'Simple',
+    `
+      description here
```
Comment (hellobrian) [inline .storybook/components/OrderSummaryStory.js]: Add a real description in all of the stories.
If there's anything specific to the react component that should be noted, also include it in the description.

**Comment by hellobrian** [comment]
image

Couple of linting errors in the tests

**Comment by tw15egan** [comment]
@hellobrian linting errors have been fixed, added descriptions

**Comment by iangfleming** [comment]
👍 ✅

**Comment by chrisdhanaraj** [comment]
Bluemix [toolchain](https://console-regional.ng.bluemix.net/devops/toolchains/6b8b7163-1ec8-4616-82fa-aa467a17af03 "storybook-toolchain-v2"): [Delivery Pipeline](https://console-regional.ng.bluemix.net/devops/pipelines/b84aaac8-8484-4a1e-902a-ebb526eb7998 "storybook-toolchain-v2") deployed [carbon-components-react](https://github.com/carbon-design-system/carbon-components-react/releases/tag/deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20170802-190905 "deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20170802-190905") to [Bluemix Design System](https://console.bluemix.net/apps/0a253845-97bd-46c9-aa04-a220aeb8ed06?env_id=ibm:yp:us-south "ibm:yp:us-south:chrisdhanaraj_org:Bluemix Design System"), including commit e2baa84386a78fd40e2ff3836b12dd4a62e1b92c

**Pull request merged**